checkov/main.py

```diff
@@ -497,6 +497,9 @@ def add_parser_args(parser: ArgumentParser) -> None:
     parser.add('--skip-cve-package',
                help='filter scan to run on all packages but a specific package identifier (denylist), You can '
                     'specify this argument multiple times to skip multiple packages', action='append', default=None)
+    parser.add('--output-baseline-as-skipped',
```

Comment:
I think it would be better to put it directly after the baseline flag, so three lines higher 🙂

Reply:
and please add it to the docs 💪 https://github.com/bridgecrewio/checkov/blob/master/docs/2.Basics/CLI%20Command%20Reference.md